common/ferrostar/src/algorithms.rs

```diff
@@ -85,13 +68,8 @@ fn get_bearing_to_next_point(
     let current = points.next()?;
     let next = points.next()?;
 
-    // This function may return negative bearing values, but we want to always normalize to [0, 360)
-    let degrees = normalize_bearing(current.geodesic_bearing(next));
-
-    Some(CourseOverGround {
-        degrees,
-        accuracy: None,
-    })
+    let degrees = Geodesic::bearing(current, next);
```

Comment:
I noticed there was a mix of `Geodesic` vs. `Haversine` calculations used throughout - is that intentional?

If you aren't super worried about accuracy Haversine will be a little quicker.

Reply:
Yes, it was at the time. Unfortunately I can't remember now the specifics, but I believe it was related to edge cases I had discovered the hard way. For example, there's a comment around here about how one or the other will panic on an exact intersection. I agree we should be able to use haversine for most cases. For things involving direct computation of distances, perhaps geodesics make more sense, since there could be a larger error over things like a cross-country route through the US.

Reply:
None of the methods should panic! 

You weren't by chance thinking of `VincentyDistance` were you? 

Vincenty is sort of between `Geodesic` and `Haversine` on the accuracy vs speed axis, but has the significant downside that it does not converge for nearly antipodal points, and thus returns an Err (but shouldn't panic).

Reply:
Yeah, it wasn't Vincenty. It was with haversine_closest_point. I had documented the issue here: https://github.com/stadiamaps/ferrostar/blob/6133f12a12a187097d1b22b1e30acbcfa5e281ac/common/ferrostar/src/algorithms.rs#L173. 

Upstream issue: https://github.com/georust/geo/issues/1084.

Someone else beat me to a PR: https://github.com/georust/geo/pull/1119. It looks like it made it into a release but I didn't get around to checking. Will see if it works ;)

Reply:
Oh right, that. I'm sorry about that. 😞 

That should be fixed as of 0.28.0, released in February.